README.md

```diff
@@ -180,11 +209,15 @@ $ knife clc server create --name 'QASrv' \
 --wait
 ```
 
-Command supports `--bootstrap` flag which allows to connect launched machine to your Chef Server installation. Only **Linux** platform is supported.
+#### Bootstrap flag
+This command supports the `--bootstrap` flag, which allows the launched machine to connect to your Chef Server installation. Only the **Linux** platform is supported.
 
-Async bootstrap variant does not require public IP access to the machine. Chef Server credentials and other parameters will be sent to the server. They will be used by Chef Client installation script during launch. Note that bootstrapping errors cancel launch operation.
+The Async bootstrap variant does not require public IP access to the machine. Chef Server credentials and other parameters will be sent to the server. They will be used by the Chef Client installation script during launch.
```

Comment:
Async is short for 'asynchronous'. Since it is not at the beginning of the sentence, you can write it with lowercase (same goes for "Sync"). You can also expand it to 'synchronous' and 'asynchronous' if you find it better.